src/main/java/com/example/java/maven/cardGamePeterPan/Player.java

```diff
@@ -0,0 +1,86 @@
+package com.example.java.maven.cardGamePeterPan;
+
+import java.util.ArrayList;
+
+public class Player {
```

Comment:
as we have a lot of logic around `hand` you can consider introducing new class: `Hand` where we will have all methods related to hand, for example: `discardCards`, `getCard`